app/services/agreements/iaa_order_seeder.rb

```diff
@@ -0,0 +1,48 @@
+module Agreements
+  class IaaOrderSeeder < BaseSeeder
+    private
+
+    def record_class
+      IaaOrder
+    end
+
+    def filename
+      'iaa_orders.yml'
+    end
+
+    def primary_attribute_bundle(config)
+      { 'iaa_gtc_id' => config['iaa_gtc'].id, 'order_number' => config['order_number'] }
+    end
+
+    def process_config(iaa_name, config)
+      match = /^(?<gtc_number>[A-Z0-9]+)-(?<order_number>\d+)$/.match(iaa_name)
```

Comment:
I am worried we are going to typo something in these configs and they're going to be skipped or error, can we make these separate fields instead?

Reply:
Unfortunately that's the unique identifier for a given IAA order, I'm not sure how else to manage it (an order number scoped to a GTC). On the flip side the config file will already be checked by the tests in the config repo before being merged so the odds of something failing on the IdP side are hopefully slimmer (although the point still stands for the YAML structure in general).

If you have any alternatives for the unique key for the YAML file let me know!

Reply:
As per Slack convo, we're adjusting the structure of this YAML file a little bit to be more explicit about those attributes (see https://github.com/18F/identity-idp-config/pull/253).